examples/typed_gen_server/lib/typed_gen_server/stage1.ex

```diff
@@ -41,27 +43,25 @@ defmodule TypedGenServer.Stage1.Server do
     end
   end
 
-  @impl true
+  # @impl true
```

Comment:
Why comment out this here and in the other places?

Reply:
This only makes sense if `use GenServer` is also enabled at the top, but this in turn leads to a type error reported within `GenServer` implementation. I don't want this to clutter the output when demoing, but ultimately we would want these uncommented and the error sorted out.

Good point, though, I'll create a ticket to track this.

Reply:
Tracked as https://github.com/esl/gradient/issues/64